src/Pos/Block/Worker.hs

```diff
@@ -112,7 +113,11 @@ blkOnNewSlotImpl (slotId@SlotId {..}) sendActions = do
         proxyCerts <- getProxySecretKeys -- TODO rename it with "light" suffix
         let validCerts =
                 filter (\pSk -> let (w0,w1) = pskOmega pSk
-                                in siEpoch >= w0 && siEpoch <= w1) proxyCerts
+                                in and [ siEpoch >= w0
+                                       , siEpoch <= w1
+                                       , not $ isRevokePsk pSk
+                                       ])
+                       proxyCerts
```

Comment:
I think it makes more sense to just prohibit putting revocation lighweight PSKs into MiscDB, but it's minor.